modules/monitoring-creating-a-monitoringstack-object-for-cluster-observability-operator.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// monitoring/cluster-observability-operator/configuring-the-cluster-observability-operator-to-monitor-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-a-monitoringstack-object-for-cluster-observability-operator_{context}"]
+= Creating a MonitoringStack object for Cluster Observability Operator
+
+To be able to discover the sample `ServiceMonitor` resource you created, you must create and configure a `MonitoringStack` object.
+
+This procedure shows how to create a `MonitoringStack` object to discover the sample `ServiceMonitor` resource for a sample service named `prometheus-coo-example-app` in the `ns1-coo` namespace.
```

Comment:
Tbh I'd maybe actually rewrite the intro a bit, and instead of the two paragraphs have something like:

"You can create a `MonitoringStack` object to discover the sample `ServiceMonitor` object.  The `ServiceMonitor` object is for a sample service named `prometheus-coo-example-app` in the `ns1-coo` namespace."

That way it:
- breaks up a long sentence
- removes ambiguity referring to a different (earlier) procedure / `ServiceMonitor` resource
- removes self-referencing from the intro

Just a suggestion though 🙂

Reply:
Good suggestion. Done